contracts/PlotManager.sol

```diff
@@ -254,6 +277,10 @@ contract PlotManager is Initializable, Ownable {
 
     // TODO: implement directions
     splitMerge.removeGeohashesFromPackage(a.packageTokenId, _geohashes, _directions1, _directions2);
+
+    if(splitMerge.getPackageCount(a.packageTokenId) == 0 && a.status == ApplicationStatuses.NEW) {
```

Comment:
should we match rejected as well?

Reply:
Documentation says: "Validator can receive a reward if rejected application have 0 geohashes in pack(disassembled)"